send_sms.py

```diff
@@ -0,0 +1,31 @@
+import requests
+import json
+import general
+
+URL = 'http://www.way2sms.com/api/v1/sendCampaign'
```

Comment:
It would make sense to move this (and other configs like apikey, secret key) to the config file https://github.com/jarifibrahim/resultBot/blob/master/config.py